kubernetes/zenko/charts/cloudserver/templates/_env.tpl

```diff
@@ -2,14 +2,10 @@
 env:
   - name: DATA_HOST
     value: "{{- printf "%s-%s" .Release.Name "s3-data" | trunc 63 | trimSuffix "-" -}}"
-  - name: REDIS_HOST
+  - name: REDIS_SENTINEL_HOST
     value: "{{- printf "%s-%s" .Release.Name "redis-ha" | trunc 63 | trimSuffix "-" -}}"
-  - name: REDIS_PORT
-    value: "6379"
-  - name: REDIS_HA_HOST
-    value: "{{- printf "%s-%s" .Release.Name "redis-ha" | trunc 63 | trimSuffix "-" -}}"
-  - name: REDIS_HA_PORT
-    value: "6379"
+  - name: REDIS_SENTINEL_PORT
```

Comment:
I spoke with @rahulreddy and suggested we should be adding this to the backbeat config as well.